tests/test_Sequence.cc

```diff
@@ -28,30 +28,49 @@
 using namespace task;
 using namespace std::literals;
 
-TEST_CASE("Sequence: Construct empty sequence", "[Sequence]")
-{
-    Sequence seq;
-    REQUIRE(seq.empty());
-    REQUIRE(seq.size() == 0);
-
-    static_assert(std::is_default_constructible<Sequence>::value,
-        "Sequence is_default_constructible");
-}
-
 TEST_CASE("Sequence: Constructor without descriptive name", "[Sequence]")
 {
     REQUIRE_THROWS_AS(Sequence(""), Error);
 }
 
 TEST_CASE("Sequence: Constructor with too long descriptive name", "[Sequence]")
 {
+    // label is to many characters -> throws Error
     REQUIRE_THROWS_AS(Sequence(std::string(Sequence::max_label_length + 1, 'c')), Error);
+    // empty label -> throws Error
+    REQUIRE_THROWS_AS(Sequence(""), Error);
+    // minimum label length with one character
+    REQUIRE_NOTHROW(Sequence("S"));
+    // label length with all characters filled
     REQUIRE_NOTHROW(Sequence(std::string(Sequence::max_label_length, 'c')));
 }
 
+TEST_CASE("Sequence: Construct an empty Sequence", "[Sequence]")
+{
+    Sequence s{ "test_sequence" };
+    REQUIRE(s.size() == 0);
+    REQUIRE(s.empty());
+    REQUIRE(s.begin() == s.end());
+    REQUIRE(s.cbegin() == s.cend());
+}
+
+TEST_CASE("Sequence: get and set sequence label", "[Sequence]")
+{
+    Sequence s{ "test_sequence" };
+    REQUIRE(s.get_label() == "test_sequence");
+
+    s.set_label("modified_test_sequence");
+    REQUIRE(s.get_label() == "modified_test_sequence");
+
+    REQUIRE_THROWS_AS(s.set_label(std::string(Sequence::max_label_length + 1, 'a'))
+        , Error);
```

Comment:
A `,` at the beginning of a line :thinking:

Reply:
I do not have any meaning about C++ source fragmentation due to long line that most be separated. In the [C++ Style Guide](https://confluence.desy.de/pages/viewpage.action?pageId=41648870) I don't find any particular hint 😕 ...

But nevertheless I have moved the comma to the previous line.